webapp/src/components/Snackbar/Snackbar.component.tsx

```diff
@@ -0,0 +1,69 @@
+import React, {useEffect, useRef} from 'react';
+import {useDispatch} from 'react-redux';
+
+import {Button} from '@brightscout/mattermost-ui-library';
+
+import {Icon} from 'components/Icon';
+
+import {getSnackbarState} from 'selectors';
+
+import usePluginApi from 'hooks/usePluginApi';
+import {closeAlert} from 'reducers/snackbar';
+import {alertTimeout} from 'constants/common.constants';
+
+import {SnackbarColor} from './Snackbar.types';
+
+export const Snackbar = () => {
+    const dispatch = useDispatch();
+    const {state} = usePluginApi();
+    const timeId = useRef(0);
+    const {isOpen, message, severity} = getSnackbarState(state);
+
+    const handleClose = () => dispatch(closeAlert());
+
+    useEffect(() => {
+        if (isOpen) {
+            timeId.current = window.setTimeout(() => {
+                // Hide the snackbar after 4 seconds
+                handleClose();
+            }, alertTimeout);
+        }
+
+        return () => {
+            clearTimeout(timeId.current);
+        };
+    }, [isOpen]);
+
+    const snackbarColorMap: Record<SnackbarColor, string> = {
+        error: 'var(--error-text)',
+        default: 'var(--center-channel-color)',
+        success: 'var(--online-indicator)',
+    };
+
+    return (
+        <div
+            className={'absolute bottom-20 right-20 left-20 py-8 px-12 rounded-4 d-flex gap-8 items-center justify-between elevation-2 msteams-sync-rhs__snackbar'}
```

Comment:
Is there a container with `position: relative` which will always consistently nest this? Would fixed position be better fit for this scenario?